src/main/java/org/apache/sling/distribution/journal/messages/PackageMessage.java

```diff
@@ -41,20 +39,42 @@ public class PackageMessage {
     String pkgBinaryRef;
     String pubAgentName;
     String userId;
-    
+
     @Builder.Default
     List<String> paths = new ArrayList<>();
-    
+
     @Builder.Default
     List<String> deepPaths = new ArrayList<>();
-    
+
+    Map<String, String> metadata;
```

Comment:
Could we initialise this with an empty map in order to avoid having to check for null ?